### PR TITLE
Reuse TurboTasks cross turbotrace runs

### PR DESCRIPTION
**Pull request by Brooooooklyn**
Allow reusing `TurboTasks` cross multiple `turbotrace` runs, can create `TurboTasks` in the next.js side and pass it back to reuse it.

Partially Close WEB-562

### CODE REVIEW

**Review by jridgewell — APPROVED** [review]

File: cli/internal/ffi/proto/messages.pb.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.28.0
```
Comment (sokra) [inline cli/internal/ffi/proto/messages.pb.go:3]: Why was this changed in this PR? Why is this file committed at all if it's generated?

**Review by sokra — CHANGES_REQUESTED** [review]

File: cli/internal/ffi/proto/messages.pb.go (hunk 1 above)
Comment (Brooooooklyn) [reply cli/internal/ffi/proto/messages.pb.go]: it seems to be changed by my `rust-analyzer`, reverted.

**Review by sokra — APPROVED** [review]
